config.json

```diff
@@ -0,0 +1,14 @@
+{

```

Comment:
Please rename this to config.json.example to avoid actually having it parsed by your code, since its' not actually safe for everyone.